planet/cli/orders.py

```diff
@@ -231,36 +233,61 @@ def read_file_json(ctx, param, value):
 @click.pass_context
 @translate_exceptions
 @coro
-@click.option('--name', required=True)
-@click.option('--id',
-              'ids',
-              help='One or more comma-separated item IDs',
-              type=click.STRING,
-              callback=split_list_arg,
-              required=True)
-# @click.option('--ids_from_search',
-#               help='Embedded data search')
+@click.argument('request',
+                required=True,
+                type=click.Path(exists=True,
+                                resolve_path=True,
+                                writable=True,
+                                allow_dash=True))
+@pretty
+async def create(ctx, request: str, pretty):
+    '''  Create an order.
+
+        This command creates an order from an order request.
+        It outputs the created order description, optionally pretty-printed.
+
+        Arguments:
+
+        Order request as stdin, str, or file name. Full description of order
+        to be created.
+    '''
+    if request == "-":
+        request_json = json.loads(sys.stdin.read())
+    if path.exists(request) and path.isfile(request):
+        request_json = json.loads(open(request).read())
```

Comment:
@kevinlacaille in the mercantile CLI I use click.open_file for this https://github.com/mapbox/mercantile/blob/main/mercantile/scripts/__init__.py#L34.